### ISSUE TITLE
Discord Player issues.

### ISSUE THREAD

**Issue by ketsunewa** [Androz2091/discord-player#156]
So, i've been coding a basic discord music bot and i've fixed atleast 2 of the hardest errors i had to fight with..

but, now i have a bigger error. And the console points to the discord-player node, the code doesn't have any problems (i think) so it has to be the node. I tried the discord player version 3.0.0, 3.0.2 & 2.3.1. Anyone have any ideas?

also; the code (the part that is supposed to be failing) & error

`try{ 
    if(command === 'play') {
        let track = await bot.player.play(message.member.voice.channel, args[0], message.member.user.tag);
        message.channel.send(`Current song: ${track.name} :white_check_mark: - Song requested by: ${track.requestedBy}`);
        return;
    } `

   ` if(command === 'stop') {
        let track = await bot.player.stop(message.guild.id);
        message.channel.send(`Song succesfully stopped. :white_check_mark:`);
        return;
    }
} catch(error) { 
  console.error(error);
}
});`

`TypeError: Cannot read property 'name' of undefined
    at new Track (C:\Users\no\Desktop\no\discord_bot\node_modules\_discord-Player_\src\Track.js:54:41)    
    at Player.play (C:\Users\no\Desktop\no\discord_bot\node_modules\_discord-Player_\src\Player.js:340:27)
    at processTicksAndRejections (internal/process/task_queues.js:93:5)
    at async Client.<anonymous> (C:\Users\no\Desktop\no\discord_bot\index.js:24:21)`

**Comment by TasoOneAsia** [comment]
Would the link by chance be a youtube one? If so, this should be fixed with this commit https://github.com/Androz2091/discord-player/commit/771ca7cebc8531e4bab7e60fb96f3f05a86a461f. To resolve this please use the master branch of this repo or wait till a new release is pushed.

**Comment by ketsunewa** [comment]
> Would the link by chance be a youtube one? If so, this should be fixed with this commit [771ca7c](https://github.com/Androz2091/discord-player/commit/771ca7cebc8531e4bab7e60fb96f3f05a86a461f). To resolve this please use the master branch of this repo or wait till a new release is pushed.

I don't send links, but say a 'name' of the song for it to search on youtube. Probably something's missing in the code block that i didn't understand. If you could tell me a piece of code that could fix it, i would be grateful

**Comment by ketsunewa** [comment]
> Would the link by chance be a youtube one? If so, this should be fixed with this commit [771ca7c](https://github.com/Androz2091/discord-player/commit/771ca7cebc8531e4bab7e60fb96f3f05a86a461f). To resolve this please use the master branch of this repo or wait till a new release is pushed.

I'm also trying the method you sent me, thanks!

**Comment by TasoOneAsia** [comment]
Here is what you do to search for a song and play it.

```javascript
// Message = Discord Message Object, 
// Query = This can be a search string, URL link, or even a Track object
client.player.play(message, query)
```

You can find all the types/parameters in the docs. For example, [here](https://discord-player.js.org/Player.html#play) is the page for that play() method we just invoked. 

Best place to find out how the player works is through docs (if you aren't looking at the source code).
